pkgs/development/python-modules/manuf/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+, wireshark-cli
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "coolbho3k";
+    repo = "manuf";
+    rev = "${version}";
+    hash = "sha256-3CFs3aqwE8rZPwU1QBqAGxNHT5jg7ymG12yBD56gTNI=";
+  };
+
+  nativeBuildInputs = [ wireshark-cli ];
+
+  prePatch = ''
```

```suggestion
  postPatch = ''
```
prePatch is really only used for operations that need to be done before the patch phase, such as dos2unix, which converts the file format.

---

Done